packages/suite/src/components/wallet/PrivacyAccount/SliderInput.tsx

```diff
@@ -3,46 +3,45 @@ import styled from 'styled-components';
 import { Input, InputProps, variables } from '@trezor/components';
 
 const LevelContainer = styled.div`
-    position: absolute;
-    right: 20px;
     width: 68px;
-
-    ${Input.InputAddon} {
```

Comment:
it was done this way to enable focusing on the input by clicking on the label. Should have left a comment 🙏

Reply:
23fb92b0655ff86054ac99b42a1330fb37f7f69d Ok, I missed it.